src/components/atoms/buttons/Button.js

```diff
@@ -103,3 +104,6 @@ export const Button = ({ children, ...other }) => (
     {children}
   </StyledButton>
 )
+Button.propTypes = {
+  children: propTypes.node,
```

Comment:
I just noticed. "propTypes.node" should start with capital **P** - "PropTypes.node"